hydpy/models/hland_v1.py

```diff
@@ -139,6 +140,19 @@
 ...         altitudefactor(-0.1)
 ...         precipitationfactor(0.1)
 
+|hland_v1| uses an optional submodel for calculating the runoff concentration. Therefore,
+we select |rconc_uh|:
```

Comment:
Maybe: "|hland_v1| can use a submodel that follows the |RConcModel_V1| to consider additional runoff concentration processes.  A possible choice is |rconc_uh|, which implements the unit hydrograph method and provides a convenient way to set the required ordinates in the triangle shape typical for HBV96:".